Lib/enum.py

```diff
@@ -414,6 +510,33 @@ def __new__(metacls, cls, bases, classdict, **kwds):
             if _order_ != enum_class._member_names_:
                 raise TypeError('member order does not match _order_')
         #
+        # remove Flag structures if final class is not a Flag
+        if Flag is None or not issubclass(enum_class, Flag):
+            delattr(enum_class, '_boundary_')
+            delattr(enum_class, '_flag_mask_')
+            delattr(enum_class, '_all_bits_')
+            delattr(enum_class, '_inverted_')
+        elif Flag is not None and issubclass(enum_class, Flag):
+            # ensure _all_bits_ is correct and there are no missing flags
+            single_bit_total = 0
+            multi_bit_total = 0
+            for flag in enum_class._member_map_.values():
+                if _is_single_bit(flag._value_):
+                    single_bit_total |= flag._value_
+                else:
+                    # multi-bit flags are considered aliases
+                    multi_bit_total |= flag._value_
+            missed = []
+            all_bit_total = 0
+            for i in range(_bit_len(max(single_bit_total, multi_bit_total))):
+                i = 2**i
+                all_bit_total |= i
+                if i & multi_bit_total and not i & single_bit_total:
+                    missed.append(i)
```

Comment:
not sure why this iteration is needed-- isn't the result of `multi_bit_total & ~single_bit_total` the "missed bits", and we can just do a bit walk of that value?

Reply:
Changed.